pywbemtools/pywbemcli/_connection_repository.py

```diff
@@ -44,6 +47,7 @@ class ConnectionRepository(object):
     _pywbemcli_servers = {}
     _loaded = False
     _connections_file = None
+    yaml_connections_group = 'connection-definitions'
```

Comment:
I suggest to name this variable independent of the format YAML or JSON.

Reply:
No problem doing this

Reply:
DONE